python/lsst/ts/wep/task/combineZernikesSigmaClipTask.py

```diff
@@ -73,26 +81,42 @@ def __init__(self, **kwargs: Any) -> None:
             self.sigmaClipKwargs[key] = val
         self.maxZernClip = self.config.maxZernClip
         self.stdMin = self.config.stdMin
+        self.zkClipType = self.config.zkClipType
+
+    def _combineZernikes(self, zkTable: Table) -> Table:
+        # Extract the array of Zernikes to perform clipping on
+        if self.zkClipType == "opd":
```

Comment:
Prefer to use `zkTable.meta["nollIndices"]` to regexes if possible.

Reply:
Ah yes, nice catch. I forgot to go back and replace these!